recipes/bioconvert/meta.yaml

```diff
@@ -1,10 +1,12 @@
+{% set version = "0.2.0" %}
+
 package:
   name: bioconvert
-  version: "0.0.10"
+  version: {{ version }}
 
 source:
-  url: https://pypi.python.org/packages/51/16/36baee3cf988d3f043e98341c43c05fd771f5dbeaef118ace23853e005d5/bioconvert-0.0.10.tar.gz
-  md5: a7073c2b80712e0884c6414eea4dee24
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
you will need to define name as a variable like you did for version, for this to work.